charts/stardog/files/utils.sh

```diff
@@ -1,19 +1,20 @@
 
 function wait_for_start {
     (
-    PORT=${1}
+    HOST=${1}
+    PORT=${2}
     # Wait for stardog to be running
     RC=1
     COUNT=0
     set +e
     while [[ ${RC} -ne 0 ]];
     do
-      if [[ ${COUNT} -gt 90 ]]; then
+      if [[ ${COUNT} -gt 300 ]]; then
           return 1;
       fi
       COUNT=$(expr 1 + ${COUNT} )
       sleep 1
-      curl -v  http://localhost:${PORT}
+      curl -v  http://${HOST}:${PORT}
```

Comment:
I think this should be:

```
curl -v http://localhost:${PORT}/admin/healthcheck
```

Credit to @buzztroll here: https://github.com/stardog-union/starctl/pull/50/files

Reply:
fixed